### PR TITLE
Remove haze

### PR DESCRIPTION
**Pull request by rreusser**
This PR removes haze from fog. I think @ansis is right that the style spec was pushed through a little too quickly. I think we all agree it looks good enough to strive to include it, but it probably warrants just a bit more thought.

This PR remove haze-color and follows through all the effects of setting fog density = 1 (which mostly reduces related operations to a no-op.)

### CODE REVIEW

File: src/render/painter.js [hunk 1]
```diff
@@ -188,14 +188,13 @@ class Painter {
...
         // We start culling where the fog opacity function hits 98%, leaving
-        // a non-noticeable opacity change threshold. We use an arbitrary function
-        // which bounds the true answer. See: https://www.desmos.com/calculator/lw03ldsuhy
-        const fogBoundFraction = 1 - 0.22 * Math.exp(4 * (fogDensity - 1));
+        // a non-noticeable opacity change threshold.
+        const fogBoundFraction = 0.78;
         const fogCullDist = fogStart + (fogEnd - fogStart) * fogBoundFraction;
 
-        this.transform.fogCullDistSq = fogCullDist * fogCullDist;
+        this.transform.fogCullDistSq = fogEnd * fogEnd;
```
Comment (karimnaaji) [inline src/render/painter.js:197]: This line should stay `this.transform.fogCullDistSq = fogCullDist * fogCullDist;`

**Review by karimnaaji — APPROVED** [review]

File: src/render/painter.js (hunk 1 above)
Comment (rreusser) [reply src/render/painter.js:197]: ack, sorry. 🤦

Comment (rreusser) [reply src/render/painter.js:197]:
```suggestion
        this.transform.fogCullDistSq = fogCullDist * fogCullDist;
```

**Comment by rreusser** [comment]
😢

**Pull request merged**